Tools/YamlCreate.ps1

```diff
@@ -38,8 +38,8 @@ if ($Settings) {
     exit
 }
 
-$ScriptHeader = '# Created with YamlCreate.ps1 v2.0.0'
-$ManifestVersion = '1.0.0'
+$ScriptHeader = '# Created with YamlCreate.ps1 v2.0.2'
```

Comment:
```suggestion
$ScriptHeader = '# Created with YamlCreate.ps1 v2.0.1'
```

Reply:
2.0.1 is a separate commit. 2.0.2 is appropriate in this case.

(Yes, I know the branch is misnamed)

Reply:
I thought it was a typo.